proposals/store.md

```diff
@@ -0,0 +1,200 @@
+Despite the principal author technically being a [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life), there are only weak and implicit power structures that are largely centered around demonstrated achievements, and decisions largely are based on technical merit as judged by those involved.
+
+Self-organised formal [community teams](https://nixos.org/community/#governance-teams) take responsibility for maintaining code in their areas of expertise, and have a more formal communication and permission structure.
+Otherwise, apart from more regular activity, they operate just like any other group of contributors.
+
+Nix itself is currently [maintained by a team of six people](https://github.com/NixOS/nix/tree/master/maintainers#members) including principal author, who meet twice a week to discuss technical decisions and review contributions.
+
+Ecosystem-wide technical decisions are negotiated in an [RFC process](https://github.com/NixOS/rfcs/).
+
+The NixOS Foundation mainly ensures funding for critical assets such as the binary cache, manages permissions in the GitHub organisation, and facilitates community activities.
+The Foundation board as a legal entity is not involved in technical decisions.
+
+Project governance has immensely gained in structure lately: In the past year, four of the eleven currently active community teams were formed, the NixOS Foundation board was restructured, and there exist plans for setting up paid support functions to ensure continued operations.
+
+## How does this project handle security risks? Are there policies, procedures, or tools in place to minimize the introduction of vulnerabilities or undesired contributions?
+
+There exists a [security team](https://nixos.org/community/teams/security.html) and an [infrastructure team](https://nixos.org/community/teams/infrastructure.html) tending to critical assets.
+[Security-relevant issues and pull-requests](https://github.com/NixOS/nixpkgs/labels/1.severity%3A%20security) are automatically tagged by a [vulnerability tracking system](https://github.com/nix-community/vulnix/).
+
+Different components have different ownership structures.
+For Nix itself, only maintainers have write access, and only the principal author can publish releases.
+Nixpkgs and NixOS have a much broader committer community with largely implicit rules.
+There is a high degree of social trust involved, and active developers and maintainers, while culturally and geographically diverse, usually have ongoing working relationships marked by diligent reviews and critical discussions.
+
+## How will you address the challenge described? Give an overview of your work and why it is significant, specifically addressing the challenge.
+
+Explain what the field will learn from your proposed work and how it contributes to the long-term sustainability of the technology.
```

Comment:
```suggestion
## How will you address the challenge described? Give an overview of your work and why it is significant, specifically addressing the challenge. Explain what the field will learn from your proposed work and how it contributes to the long-term sustainability of the technology.
```

@iFreilicht Thanks for catching that. It's an artifact of splitting lines by sentence for easier reviews and suggestions.